modules/ROOT/pages/dataweave-types.adoc

```diff
@@ -733,7 +733,10 @@ var y = {e: "e"}
 }
 ----
 
-It is important to note that the expression between the _parentheses_ (such as `(x)` where `x` is the variable shown in the header) should return an array of objects. All of objects in that array get merged together. They are also merged with the contained object. So the output looks like this:
+The _evaluation parentheses_ around the variables in the body of the script above (`(x)` and `(y)`) enable the object constructor curly braces (`{ }`) surrounding the body of the script to extract the key-value pairs from the values of `x` and `y` and convert those values into a collection of key-value pairs within an object. Without the parentheses (for example, if you use `x` instead of `(x)`), the script returns an error `Invalid input '}', expected Namespace or Attribute&lt;'@('(Name:Value)+')'&gt;`.
```

Comment:
I would just delete the error message because that could change over time. Like this: `Without the parentheses (for example, if you use `x` instead of `(x)`), the script returns an error.`
The rest looks correct for me.

Reply:
done.